tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -31,7 +31,8 @@ const createNewFolderButton = '#new-folder-btn'
 const createNewTxtFileButton = '.new-file-btn-txt'
 const createNewMdFileButton = '.new-file-btn-md'
 const createNewDrawioFileButton = '.new-file-btn-drawio'
-const saveTextFileInEditorButton = '#text-editor-controls-save'
+const createNewOpenDocumentFileButton = '//ul[@id="create-list"]//li//button/span[text()="%s"]'
```

Comment:
```suggestion
const externalCreateNewFileButton = '//ul[@id="create-list"]//li//button/span[text()="%s"]'
```

Reply:
can't we use `button/span[text()="%s"` as id selector

Reply:
good to have the container as well. It will scope the selector